src/rgw/rgw_common.h

```diff
@@ -151,10 +151,33 @@ using ceph::crypto::MD5;
 
 #define RGW_ATTR_TRACE RGW_ATTR_PREFIX "trace"
 
-#define RGW_FORMAT_PLAIN        0
-#define RGW_FORMAT_XML          1
-#define RGW_FORMAT_JSON         2
-#define RGW_FORMAT_HTML         3
+enum class RGWFormat : int8_t {
+  BAD_FORMAT = -1,
+  PLAIN = 0,
+  XML,
+  JSON,
+  HTML,
+};
+
+static inline const char* toString(const RGWFormat f)
```

Comment:
`to_string()` functions are a good way to associate names with the values of an enum class

in this case though, the strings we want are not just enum names but MIME types. could we call this function `to_mime_type()` instead?

alternatively, we could rename the enum from `RGWFormat` to `RGWMimeType` so it's more obvious what `to_string()` will give you - but `RGWFormat` makes sense in that it controls which kind of `ceph::Formatter` we construct